source/app/blueprints/rest/manage/manage_cases_routes.py

```
@@ -14,15 +14,22 @@
 #  You should have received a copy of the GNU Lesser General Public License
 #  along with this program; if not, write to the Free Software Foundation,
 #  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-
+from typing import Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)